build.sbt

```diff
@@ -18,27 +16,27 @@ dependencyOverrides ++= Set(
   "com.jcraft"                        %  "jsch"                        % "0.1.51"
 )
 
+crossSbtVersions := Vector("0.13.16", "1.0.0")
+
 libraryDependencies ++= Seq (
-  "com.fasterxml.jackson.core"        %  "jackson-core"                % "2.6.1",
-  "com.fasterxml.jackson.module"      %% "jackson-module-scala"        % "2.6.1",
+  "com.fasterxml.jackson.core"        %  "jackson-core"                % "2.9.0",
+  "com.fasterxml.jackson.module"      %% "jackson-module-scala"        % "2.9.0",
   // DO NOT UPGRADE: later versions of jgit use Java 7 and we still need to support Java 6
```

Comment:
Note: since SBT 1.0 already depends on JDK 8, it should be safe to upgrade to the latest JGit version now.

Reply:
at the moment, it supports 0.13 and 1.0, which means we still probably want jdk7 for 0.13 version

Reply:
I think it's not a big delay not to upgrade the dependencies, the top priority should be the sbt 1.0 support ;)